source/includes/_payment.md

```diff
@@ -2769,9 +1724,6 @@ class Payment {
    "auditLogs":[]
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?